core/src/main/scala/org/bitcoins/core/protocol/blockchain/BlockHeader.scala

```diff
@@ -125,7 +125,7 @@ sealed trait BlockHeader extends NetworkElement {
     * [[https://bitcoin.stackexchange.com/questions/2063/why-does-the-bitcoin-protocol-use-the-little-endian-notation]]
     * @return
     */
-  def hashBE: DoubleSha256DigestBE = hash.flip
+  lazy val hashBE: DoubleSha256DigestBE = hash.flip
```

Comment:
I don't think this is an actual optimization because `DoubleSha256Digest.flip` is already a `lazy val`, but good to do this anways

Reply:
It is, it showing up on visualvm as ~1% of cpu usage when syncing filters